modules/microshift-cri-o-container-runtime.adoc

```diff
@@ -6,18 +6,31 @@
 [id="microshift-CRI-O-container-engine_{context}"]
 = Using a proxy in the CRI-O container runtime
 
-To use an HTTP(S) proxy in `CRI-O`, you need to set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. You can also set the `NO_PROXY` variable to exclude a list of hosts from being proxied.
+To use an HTTP(S) proxy in `CRI-O`, you must add a `Service` section to the configuration file and set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables. You can also set the `NO_PROXY` variable to exclude a list of hosts from being proxied.
 
 .Procedure
 
+. Create the directory for the configuration file if it does not exist:
+
+[source,terminal]
+----
+$ mkdir /etc/systemd/system/crio.service.d/
+----
+
 . Add the following settings to the `/etc/systemd/system/crio.service.d/00-proxy.conf` file:
 +
 [source,config]
 ----
+[Section]
```

Comment:
Should this say `Service` or `Section` ?

Reply:
Service, yeah, my mistake in typing.